content/rs/developing/crdbs/developing-streams-crdb.md

```diff
@@ -151,12 +151,12 @@ Here is an example of a simple XGROUP case:
 | t4   | --- sync ---                | --- sync ---                |
 | t5   | XINFO GROUPS x --> [g1, g2] | XINFO GROUPS x --> [g1, g2] |
 
-Here is an example of a concurrent XADD case:
+Here is an example of a concurrent XADD case: // There is no XADD in this example.... (and I think there should be!)
```

Comment:
I changed this line to reflect the example in https://github.com/RedisLabs/redislabs-docs/pull/813